package.json

```diff
@@ -101,6 +100,7 @@
     "npm-run-all": "^4.0.2",
     "npmpub": "^3.1.0",
     "postcss-import": "^11.0.0",
+    "postcss-markdown": "^0.30.0",
```

Comment:
@gucong3000 Should `postcss-markdown` have moved from `dependencies` to `devDependencies`?